src/http.js

```diff
@@ -72,9 +72,10 @@ export default class HTTP {
    * @param  {String} url               The URL.
    * @param  {Object} [options={}]      The fetch() options object.
    * @param  {Object} [options.headers] The request headers object (default: {})
+   * @param  {Object} [options.retry]   Number of retries (default: 1)
    * @return {Promise}
    */
-  request(url, options={headers:{}}) {
+  request(url, options={headers:{}, retry: 1}) {
```

Comment:
According to https://github.com/Kinto/kinto-http.js/pull/155/commits/915c6b152b402e9032553324f20342ca2260b117 should it be 0 by default now?